### PR TITLE
DAC6-118 - What is your Date of Birth? page rework

### PR DESCRIPTION
**Pull request by soljeggers**

### CODE REVIEW

File: test/helpers/DateHelperSpec.scala [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+      "must return '1 January 2020' if today's date is 2020-1-1" in {
+
+        DateHelper.setDate(Some(LocalDate.of(2020,1,1)))
+
+        DateHelper.formatDateToString(DateHelper.today) mustBe "1 January 2020"
+
+      }
+
+      "must return '5 Jun 1977' if given past date 1977-6-5" in {
```
Comment (daniel-manning) [inline test/helpers/DateHelperSpec.scala:37]: not right description - short form date

File: app/forms/DateOfBirthFormProvider.scala [hunk 2]
```diff
@@ -31,6 +32,8 @@ class DateOfBirthFormProvider @Inject() extends Mappings {
         allRequiredKey = "dateOfBirth.error.required.all",
         twoRequiredKey = "dateOfBirth.error.required.two",
         requiredKey    = "dateOfBirth.error.required"
-      )
+      ).verifying(maxDate(today, "dateOfBirth.error.futureDate", formatDateToString(today)))
+       .verifying(minDate(LocalDate.of(1900,1,1),"dateOfBirth.error.pastDate"))
```
Comment (daniel-manning) [inline app/forms/DateOfBirthFormProvider.scala:36]: why?

Comment (soljeggers) [reply app/forms/DateOfBirthFormProvider.scala:36]: why are we adding additional validation?

Comment (daniel-manning) [reply app/forms/DateOfBirthFormProvider.scala:36]: where is this requirement coming from?

Comment (soljeggers) [reply app/forms/DateOfBirthFormProvider.scala:36]: I'd assume it came from the back of initial testing when the page was originally created then went through design?

would we want to allow users to put a future date in for a date of birth? or a date that is too far in the past?

Comment (daniel-manning) [reply app/forms/DateOfBirthFormProvider.scala:36]: It's fine as long as its standard across dates entered and cleared with everyone, the granny in the shed could be over 120 years old.

Comment (soljeggers) [reply app/forms/DateOfBirthFormProvider.scala:36]: hahaha.. yeah Maddie checked with content team. The last granny born in the 1800's died in 2017. God rest her soul.

**Pull request merged**